src/interface.jl

```diff
@@ -2,58 +2,70 @@
 
 
 """
-    isdensitytype(type::Type)
+    hasdensity(d)::Bool
 
-Returns `true` if `type` is compatible with the `DensityInterface` interface,
-otherwise returns `false` by default.
+Returns `true` if `d` is compatible with the `DensityInterface` interface.
```

Comment:
According to the Julia style guide for docstrings:
```suggestion
Return `true` if `d` is compatible with the `DensityInterface` interface.
```